src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -0,0 +1,527 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  Form,
+  Button,
+  FormGroup,
+  InlineNotification,
+  ComposedModal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  TextInput,
+  Toggle
+} from 'carbon-components-react';
+import {
+  PipelinesDropdown,
+  ServiceAccountsDropdown,
+  NamespacesDropdown
+} from '..';
+import {
+  getPipelineRunsErrorMessage,
+  isFetchingPipelineRuns,
+  getSelectedNamespace
+} from '../../reducers';
+import { createPipelineRun } from '../../actions/pipelineRuns';
+
+import './CreatePipelineRun.scss';
+import { ALL_NAMESPACES } from '../../constants';
+
+const resourceNameRegExp = /^[-.a-z1-9]{1,253}$/;
+const resourceNameInvalidText =
+  'Must consist of only lower case alphanumeric characters, -, and . with at most 253 characters';
+const noSpacesRegExp = /^\S*$/;
+const noSpacesInvalidText = 'Must contain no spaces';
+const urlRegExp = /^(http)s?(:\/\/).*$/;
```

Comment:
no need for the capturing groups

```diff
-const urlRegExp = /^(http)s?(:\/\/).*$/;
+const urlRegExp = /^https?:\/\/.*$/;
```

This will allow a 'URL' such as `http://` or `https://`, but changing the kleene star to a plus would probably give _slightly_ more useful validation by enforcing at least _some_ content after the //

`const urlRegExp = /^https?:\/\/.+/;`